test/adapter.js

```diff
@@ -104,6 +97,23 @@ describe('Swagger 2.0 adapter', () => {
     });
   });
 
+  context('cannot parse invalid Swagger YAML', () => {
+    const source = 'swagger: "2.0"\nbad: }';
+    let result;
+
+    before((done) => {
+      fury.parse({source}, (err) => {
+        result = err;
```

Comment:
Is the correct naming of this variable? Wouldn't the result of the parse be called the `result`, but you are concerned with the `err` here?